packages/client/src/Client.ts

```diff
@@ -70,6 +67,10 @@ export interface Client extends SharedClient, OldSharedClient {
     o: Q,
   ): QuerySignatureFromDef<Q>;
 
+  <Q extends Experiment<"2.0.8">>(
```

Comment:
Are we going to need to update these for every cut?

Reply:
Every time we add new experimental APIs or take them away.

Reply:
I see, so if in 2.0.12 we add an experimental API, we will need to change this to 2.0.12, but are we also going to need to update the rest of the API's to the new version to ensure compatibility?

Reply:
We would make this `Experiment<"2.0.8"> | Experiment<"2.0.12">`